apps/cloudfront-functions/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "cloudfront-functions",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "clean": "shx rm -rf dist/",
+    "compile": "npm run clean && tsc",
+    "lint": "eslint src",
+    "test": "npm run compile && jest -i"
+  },
+  "devDependencies": {
+    "@types/aws-cloudfront-function": "^1.0.2",
+    "@types/jest": "^29.5.11",
+    "@types/rewire": "^2.5.28",
+    "eslint": "^8.40.0",
+    "eslint-config-custom": "*",
+    "jest": "^29.5.0",
```

Comment:
Since in the `nextjs-website` app we used `vitest`, I'd use `vitest` here as well